src/components/Navigation/index.js

```diff
@@ -0,0 +1,3 @@
+import Navigation from './Navigation';
+
+export default Navigation;
```

Comment:
I'm a bit confused on why each page/component we add an index.js. The index.js pages aren't adding any sort of wrapper functionality to these components, so we could just directly import the component or page. 
In App.js we can change our imports from:
`import Navigation from '../../components/Navigation';`
to:
`import Navigation from '../../components/Navigation/Navigation';`
If I'm missing something please let me know!